core/server/master/src/main/java/alluxio/master/meta/checkconf/ServerConfigurationChecker.java

```diff
@@ -111,6 +123,7 @@ public ServerConfigurationChecker(ServerConfigurationStore masterStore,
     mMasterStore = masterStore;
     mWorkerStore = workerStore;
     mConfigCheckReport = new ConfigCheckReport();
+    mLastLogConfigReportTimeMs = System.currentTimeMillis();
```

Comment:
initialize to -1 since we haven't logged a report yet?

Reply:
The default config status is PASSED, so if we set it to -1, we will immediately log the config report which will show PASSED which I think maybe useless.
Set it to be the current time, then if the config status changes or 1h interval reached, it will log the report.

Reply:
this is hacky because the field will be incorrect for some time about the last time we logged the config report

I wonder if this time gap is more trouble than its worth. The user can already see the latest report in the UI/CLI. The logging is more for us when we collect the user logs. For that purpose, we just need to see the status changes and the hourly log report.

Reply:
Agreed! We can directly use the log heartbeat to log the config report hourly and do not check the log gap. I don't think the log gap will bring a great difference.